src/SmartComponents/SystemInventory/SystemInventoryCard.js

```diff
@@ -8,31 +15,107 @@ import messages from '../../Messages';
 /**
  * System inventory card for showing system inventory and status.
  */
-const SystemInventoryCard = () => {
+const SystemInventoryCard = ({
+    fetchInventory, inventoryFetchStatus, inventorySummary,
+    fetchInventoryStale, inventoryStaleFetchStatus, inventoryStaleSummary,
+    fetchInventoryWarning, inventoryWarningFetchStatus, inventoryWarningSummary,
+    fetchInventoryTotal, inventoryTotalFetchStatus, inventoryTotalSummary
+}) => {
+
+    useEffect(() => {
```

Comment:
Yah can just do these all on one, also, an empty array at the end will ensure this happens once, on component instantiation, (like didmount)

Reply:
empty array at the end gave me an error, so I just moved them all into one